US County Data - pandas.ipynb

```diff
@@ -82,11 +286,183 @@
+      "2020-03-05       4     NaN\n",
+      "2020-03-06       5     1.0\n",
+      "2020-03-07       5     0.0\n",
+      "2020-03-08       6     1.0\n",
+      "2020-03-09       6     0.0\n",
+      "...            ...     ...\n",
+      "2021-09-29  554336  2547.0\n",
+      "2021-09-30  555663  1327.0\n",
+      "2021-10-01  556159   496.0\n",
+      "2021-10-02  558282  2123.0\n",
+      "2021-10-03  558618   336.0\n",
+      "\n",
+      "[578 rows x 2 columns]\n",
+      "              Date  Dallas\n",
+      "date                      \n",
+      "2020-03-10       2     NaN\n",
+      "2020-03-11       3     1.0\n",
+      "2020-03-12       8     5.0\n",
+      "2020-03-13       9     1.0\n",
+      "2020-03-14      11     2.0\n",
+      "...            ...     ...\n",
+      "2021-09-29  386543  2292.0\n",
+      "2021-09-30  387470   927.0\n",
+      "2021-10-01  388413   943.0\n",
+      "2021-10-02  389318   905.0\n",
+      "2021-10-03  389318     0.0\n",
+      "\n",
+      "[573 rows x 2 columns]\n",
+      "              Date  San Diego\n",
+      "date                         \n",
+      "2020-02-10       1        NaN\n",
+      "2020-02-11       1        0.0\n",
+      "2020-02-12       1        0.0\n",
+      "2020-02-13       1        0.0\n",
+      "2020-02-14       1        0.0\n",
+      "...            ...        ...\n",
+      "2021-09-29  372208      618.0\n",
+      "2021-09-30  372734      526.0\n",
+      "2021-10-01  373289      555.0\n",
+      "2021-10-02  373289        0.0\n",
+      "2021-10-03  373289        0.0\n",
+      "\n",
+      "[602 rows x 2 columns]\n",
+      "              Date  Riverside\n",
+      "date                         \n",
+      "2020-03-07       1        NaN\n",
+      "2020-03-08       1        0.0\n",
+      "2020-03-09       4        3.0\n",
+      "2020-03-10       4        0.0\n",
+      "2020-03-11       4        0.0\n",
+      "...            ...        ...\n",
+      "2021-09-29  362691      567.0\n",
+      "2021-09-30  363281      590.0\n",
+      "2021-10-01  363950      669.0\n",
+      "2021-10-02  363950        0.0\n",
+      "2021-10-03  363950        0.0\n",
+      "\n",
+      "[576 rows x 2 columns]\n",
+      "              Date  San Bernardino\n",
+      "date                              \n",
+      "2020-03-15       1             NaN\n",
+      "2020-03-16       2             1.0\n",
+      "2020-03-17       3             1.0\n",
+      "2020-03-18       5             2.0\n",
+      "2020-03-19       5             0.0\n",
+      "...            ...             ...\n",
+      "2021-09-29  350976           390.0\n",
+      "2021-09-30  351423           447.0\n",
+      "2021-10-01  351963           540.0\n",
+      "2021-10-02  351963             0.0\n",
+      "2021-10-03  351963             0.0\n",
+      "\n",
+      "[568 rows x 2 columns]\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "counties = []\n",
+    "for i in range (10):\n",
```

Comment:
Not sure why you decided to make a list here, you should be able to do this with just the dataframe you created above.